keyboards/skyloong/dt40/info.json

```diff
@@ -0,0 +1,98 @@
+    },
+
+
+    "matrix_pins": {
+        "cols": ["B12", "B5", "B8", "B9", "B10", "B11", "A15", "B7", "A2", "A1", "A0"],
+        "rows": ["B6", "B15", "B2", "B14"]
+    },
+
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0.25, "y": 0 },
+                { "matrix": [0, 1], "x": 1.5, "y": 0 },
+                { "matrix": [0, 2], "x": 2.5, "y": 0 },
+                { "matrix": [0, 3], "x": 3.5, "y": 0 },
+                { "matrix": [0, 4], "x": 4.5, "y": 0 },
+                { "matrix": [0, 5], "x": 5.5, "y": 0 },
+                { "matrix": [0, 6], "x": 6.5, "y": 0 },
+                { "matrix": [0, 7], "x": 7.5, "y": 0 },
+                { "matrix": [0, 8], "x": 8.5, "y": 0 },
+                { "matrix": [0, 9], "x": 9.5, "y": 0 },
+                { "matrix": [0, 10], "x":11.75, "y": 0 },
+
+                { "matrix": [1, 0], "x": 0.5, "y": 1 },
+                { "matrix": [1, 1], "x": 1.75, "y": 1 },
+                { "matrix": [1, 2], "x": 2.75, "y": 1 },
+                { "matrix": [1, 3], "x": 3.75, "y": 1 },
+                { "matrix": [1, 4], "x": 4.75, "y": 1 },
+                { "matrix": [1, 5], "x": 5.75, "y": 1 },
+                { "matrix": [1, 6], "x": 6.75, "y": 1 },
+                { "matrix": [1, 7], "x": 7.75, "y": 1 },
+                { "matrix": [1, 8], "x": 9.75, "y": 1 },
+                { "matrix": [1, 9], "x": 10.5, "y": 0 },
+                { "matrix": [1, 10], "x": 11.5, "y": 1 },
+
+                { "matrix": [2, 0], "x": 0.75, "y": 2 },
+                { "matrix": [2, 1], "x": 2.25, "y": 2 },
+                { "matrix": [2, 2], "x": 3.25, "y": 2 },
+                { "matrix": [2, 3], "x": 4.25, "y": 2 },
+                { "matrix": [2, 4], "x": 5.25, "y": 2 },
+                { "matrix": [2, 5], "x": 6.25, "y": 2 },
+                { "matrix": [2, 6], "x": 7.25, "y": 2 },
+                { "matrix": [2, 7], "x": 8.25, "y": 2 },
+                { "matrix": [2, 8], "x": 9.55, "y": 2 },
+                { "matrix": [2, 9], "x": 11, "y": 2 },
+                { "matrix": [2, 10], "x": 12, "y": 2 },
+
+                { "matrix": [3, 0], "x": 0, "y": 3 },
+                { "matrix": [3, 1], "x": 1.25, "y": 3 },
+                { "matrix": [3, 2], "x": 2.25, "y": 3 },
+                { "matrix": [3, 3], "x": 3.3, "y": 3 },
+                { "matrix": [3, 4], "x": 5.5, "y": 3 },
+                { "matrix": [3, 5], "x": 8.75, "y": 1 },
+                { "matrix": [3, 6], "x": 7.5, "y": 3 },
+                { "matrix": [3, 7], "x": 8.75, "y": 3 },
+                { "matrix": [3, 8], "x": 10, "y": 3 },
+                { "matrix": [3, 9], "x": 11, "y": 3 },
+                { "matrix": [3, 10], "x": 12, "y": 3 }
+
+            ]
+        }
+    },
+
+
+
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "B4"
+    },
+    "rgb_matrix":{
+        "driver":"WS2812",
+        "max_brightness": 200,
+        "sleep": true
```

Comment:
```suggestion
        "max_brightness": 200
```

Reply:
I want the RGB matrix to synchronize sleep when the keyboard sleeps. Unsetting sleep in info.json, does it affect this feature? @waffle87